extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php

```diff
@@ -55,43 +56,39 @@ public function getInsightsModules() {
 	 * @param string $type type of module we want to build.
 	 * @param array $config array with variables that determine
 	 *  - should display page views on list
-	 * 	- how data should be sorted (@see InsightsSorting::$sorting)
+	 *  - how data should be sorted (@see InsightsSorting::$sorting)
 	 * @return array Insight Module
 	 */
 	private function getInsightModule( $type, $config ) {
 		$insightPages = $this->insightsService->getInsightPages(
 			$type,
-			self::INSIGHTS_MODULE_ITEMS,
-			$config[self::INSIGHTS_CONFIG_SORT_TYPE_KEY]
+			static::INSIGHTS_MODULE_ITEMS,
+			$config[ static::INSIGHTS_CONFIG_SORT_TYPE_KEY ]
 		);
 
-		if ( empty( $insightPages['pages'] ) ) {
-			return [];
+		if ( empty( $insightPages[ 'pages' ] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "pages" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)